src/FormCollection/README.md

```diff
@@ -0,0 +1,141 @@
+# UX Form Collection
+
+Symfony UX Form collection is a Symfony bundle providing light UX for collection
+in Symfony Forms.
+
+## Installation
+
+UX Form Collection requires PHP 7.2+ and Symfony 4.4+.
+
+Install this bundle using Composer and Symfony Flex:
+
+```sh
+composer require symfony/ux-form-collection
+
+# Don't forget to install the JavaScript dependencies as well and compile
+yarn install --force
+yarn encore dev
+```
+
+Also make sure you have at least version 2.0 of [@symfony/stimulus-bridge](https://github.com/symfony/stimulus-bridge)
+in your `package.json` file.
+
+You need to select the right theme from the one you are using :
+```yaml
+# config/packages/twig.yaml
+twig:
+  # For bootstrap for example
+  form_themes: ['@FormCollection/form_theme_div.html.twig']
+```
+You have 2 different themes :
+- `@FormCollection/form_theme_div.html.twig`
+- `@FormCollection/form_theme_table.html.twig`
+
+[Check the Symfony doc](https://symfony.com/doc/4.4/form/form_themes.html) for the different ways to set themes in Symfony.
+
+## Usage
+
+The most common usage of Form Collection is to use it as a replacement of
+the native CollectionType class:
+
+```php
+// ...
+use Symfony\UX\FormCollection\Form\CollectionType;
+
+class BlogFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            // ...
+            ->add('comments', CollectionType::class, [
+                // ...
+                'button_add' => [
+                     // Default text for the add button
+                    'text' => 'Add',    
+                    // Default attr class for the add button
+                    'attr' => ['class' => 'btn btn-outline-primary'] 
+                ],
+                'button_delete' => [
+                    // Default text for the delete button
+                    'text' => 'Remove',    
+                    // Default class for the delete button
+                    'attr' => ['class' => 'btn btn-outline-secondary']    
+                ],
+            ])
+            // ...
+        ;
+    }
+
+    // ...
+}
+```
```

Comment:
You should also show how to render it (I know, it's just "rendering the field", but people will still wonder)